tests/platform_helper/utils/test_aws.py

```diff
@@ -601,6 +604,134 @@ def test_update_postgres_parameter_with_master_secret():
     }
 
 
+def test_get_redis_supported_versions_when_cache_refresh_required():
+
+    list_redis_versions_response = [
+        {
+            "Engine": "redis",
+            "EngineVersion": "4.0.10",
+            "CacheParameterGroupFamily": "redis4.0",
+            "CacheEngineDescription": "Redis",
+            "CacheEngineVersionDescription": "redis version 4.0.10",
+        },
+        {
+            "Engine": "redis",
+            "EngineVersion": "5.0.6",
+            "CacheParameterGroupFamily": "redis5.0",
+            "CacheEngineDescription": "Redis",
+            "CacheEngineVersionDescription": "redis version 5.0.6",
+        },
+        {
+            "Engine": "redis",
+            "EngineVersion": "6.0",
+            "CacheParameterGroupFamily": "redis6.x",
+            "CacheEngineDescription": "Redis",
+            "CacheEngineVersionDescription": "redis version 6.0.5",
+        },
+        {
+            "Engine": "redis",
+            "EngineVersion": "6.2",
+            "CacheParameterGroupFamily": "redis6.x",
+            "CacheEngineDescription": "Redis",
+            "CacheEngineVersionDescription": "redis version 6.2.6",
+        },
+        {
+            "Engine": "redis",
+            "EngineVersion": "7.0",
+            "CacheParameterGroupFamily": "redis7",
+            "CacheEngineDescription": "Redis",
+            "CacheEngineVersionDescription": "redis version 7.0.7",
+        },
+        {
+            "Engine": "redis",
+            "EngineVersion": "7.1",
+            "CacheParameterGroupFamily": "redis7",
+            "CacheEngineDescription": "Redis",
+            "CacheEngineVersionDescription": "redis version 7.1.0",
+        },
+    ]
```

Comment:
For the purposes of the tests, do we need this many versions? Or can we slim it down so there is a bit less to grok?